src/Target.h

```diff
@@ -258,6 +258,16 @@ struct Target {
     /** Was libHalide compiled with support for this target? */
     EXPORT bool supported() const;
 
+    /** Given a size (in bytes), return True if the allocation size can fit
```

Comment:
I should add that one reason I am sensitive to this is because Target.h sits on the boundary between being in the compiler and in the runtime. I.e. there are cases where one wants to be able to parse and manipulate target strings in the runtime. The more of this type of functionality we roll into it, the further we get from being able to do that.